pkg/controller/networkpolicyrecommendation/controller_test.go

```diff
@@ -663,3 +694,82 @@ func TestGetPolicyRecommendationResult(t *testing.T) {
 		})
 	}
 }
+
+func initTestObjects(t *testing.T, stopCh chan struct{}) *NPRecommendationController {
```

Comment:
Maybe remove the code in this commit for controller unit test. It seems these unit tests overlap with the existed ones and do not increase the coverage.

Reply:
Yes, they are duplicated. I've removed it. Thanks.